docs/buildah-manifest-add.md

```diff
@@ -0,0 +1,86 @@
+# buildah-manifest-add "1" "September 2019" "buildah"
+
+## NAME
+
+buildah\-manifest\-add - Add an image to a manifest list or image index.
+
+## SYNOPSIS
+
+**buildah manifest add** *listName* *imageName*
+
+## DESCRIPTION
+
+Adds the specified image to the specified manifest list or image index.
+
+## RETURN VALUE
+
+The list image's ID and the digest of the image's manifest.
+
+## OPTIONS
+
+**--all**
+
+If the image which should be added to the list or index is itself a list or
+indexes, add all of its contents to the local list.  By default, only one image
+from such a list will be added to the list or index.  Combining *--all* with
+any of the other options described below is NOT recommended.
+
+**--annotation** *annotation=value*
+
+Set an annotation on the entry for the newly-added image.
+
+**--arch**
+
+Override the architecture which the list or index records as a requirement for
+the image.  If *imageName* refers to a manifest list or image index, the
+architecture information will be retrieved from it.  Otherwise, it will be
+retrieved from the image's configuration information, so it is rarely necessary
```

Comment:
I'm not a big fan of the "rarely used" bit.  I think it's fine for a blog, but not a man page.  YMMV.

Reply:
Dropping "so it is rarely necessary to use this option".  If there's a better phrasing to describe that I don't tend to see some of these fields when I point `skopeo inspect --raw` at lists in registries, I'm happy to change instances of "This option is rarely used" to something else.